### PR TITLE
Update Filter_Expressions.md

### PR DESCRIPTION
**Pull request by ScottPatterson**
Removing references to filtering on SDS objects.
https://osisoft.visualstudio.com/QiLibraries/_workitems/edit/271428

### CODE REVIEW

File: Documentation/SequentialDataStore/Filter_Expressions.md [hunk 1]
```diff
@@ -5,9 +5,7 @@ uid: sdsFilterExpressions
 Filter Expressions: Values
 ==================
 
-Filter expressions can be applied to certain read operations that return Sequential Data Store values or objects. This section covers data values. For filtering on SDS objects refer to [Filter Expressions: SDS Objects](xref:sdsFilterExpressionsObjects).
-
-Filter expressions can be applied to any read that returns multiple values, including 
+Filter expressions can be applied to certain read operations that return Sequential Data Store values including
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Filter_Expressions.md]: Add comma after "values", and colon after "including"

Comment (osimarkrogers) [reply Documentation/SequentialDataStore/Filter_Expressions.md]: Did you intend to remove link from original material?

Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Filter_Expressions.md]: I did intend to remove the link - we're removing that type of filtering for GA.

Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Filter_Expressions.md]: Grammar fixed in commit 3385837317

**Pull request merged**